scheduler/crates/api/src/shared/auth/route_guards.rs

```diff
@@ -25,11 +25,7 @@ struct Claims {
 }
 
 fn parse_authtoken_header(token_header_value: &str) -> String {
-    token_header_value
-        .replace("Bearer", "")
-        .replace("bearer", "")
-        .trim()
-        .to_string()
+    token_header_value.trim()[7..].to_string()
```

Comment:
This is going to panic if a user sends a request with an authorization header with length less than 7

Reply:
done :)